### DISCUSSION TITLE
Push notifications for GitHub website

### DISCUSSION THREAD

**Discussion by aleksey-hoffman** [community/community#5568] [Other Feature Feedback, Questions, & Ideas]
Hey guys, could you please implement push notifications for Github desktop website? It would be really convinient to know when you got a notification on Github, without checking the website

**Comment by filiptronicek** [comment]
@aleksey-hoffman perhaps this could be more helpful in https://github.com/desktop/desktop 👀.

**Reply by aleksey-hoffman** [reply]
Thanks for the suggestion. Though I'm not sure I'd want to have a desktop app opened at all times just for the notifications.

**Comment by donno2048** [comment]
You can use https://github.com/sindresorhus/notifier-for-github even though it would be better to have that built-in

**Reply by aleksey-hoffman** [reply]
Thank you, I will try it
